book/schedule.yaml

```diff
@@ -1,38 +1,171 @@
 timezone: UTC -7 (Pacific Daylight Time)
 tabs:
-- title: Day 1
-  date: Mon 11, July
-  contents:
-  - time: 9:00 - 9:30
-    location: Conference Room A
-    title: Best Practices And Tips For Research Software
-    description: A talk about reproducible research
-    leads:
-    - Scott Henderson
-    - Anthony Arendt
-    resources:
-      - title: "SnowEx Hackweek 2021: Introductory Session"
-        url: https://youtu.be/cHQloxZFtOQ
-  - time: 9:30 - 10:00
-    location: Room A5
-    title: Coffee Break
-    description: Coffee and cakes lorem ipsum dolor sit amet, consectetur adipiscing
-      elit.
-    leads: []
-- title: Day 2
-  date: Tues 12, July
-  contents:
-  - time: 1:00 - 2:00 PM
-    title: Coffee Break
-    description: |
-      
-      more refueling, but with a bulleted list:
-
-      - list1 
-      - list2
-- title: Day 3
-  date: Wed 13, July
-- title: Day 4
-  date: Thurs 14, July
-- title: Day 5
-  date: Friday 17, July
+  - title: Day 1
+    date: Monday 11 Jul
+    contents:
+      - time: 9:00 - 10:00
+        location: Maple Hall
+        title: Welcome to SnowEx Hackweek
+        description: Getting to know each other and setting the stage for our work together.
+        leads:
+          - Anthony Arendt
+          - Charley Haley
+      - time: 10:00 - 10:30
+        location: Maple Hall
+        title: Overview of the SnowEx Mission
+        description:
+        leads:
+          - HP Marshall
+      - time: 10:30 - 10:45
+        title: Movement and coffee break
+        description:
+        leads: [ ]
```

Comment:
The template is setup to handle time slots with no leads. With that, there is no need to give an empty list and the key can be removed.